.dockerignore

```diff
@@ -0,0 +1,3 @@
+/frontend/node_modules/
+/frontend/bundle.*
```

Comment:
We're not using this path pattern yet in the build. I think it comes later.

Reply:
yes sorry!

Reply:
Done.